fuzzers/strings_fuzzer.c

```diff
@@ -28,7 +29,6 @@ LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
   memcpy(ns, data, size);
   ns[size] = '\0';
 
-  pcmk_numeric_strcasecmp(ns, ns);
```

Comment:
The function was renamed with the first underbar doubled (`pcmk__numeric_strcasecmp`). In Pacemaker a single first underbar means public API and a double means internal-only, and for the next upstream release we're pulling a lot of the existing API internal.

I"ll have to remember to start checking the fuzzers when we make these changes ...

Reply:
Great, thanks! Fixed